backup_and_restore/application_backup_and_restore/oadp-rosa/oadp-rosa-backing-up-applications.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+[id="oadp-rosa-backing-up-applications"]
+= Backing up applications ROSA clusters using OADP
+include::_attributes/common-attributes.adoc[]
+:context: oadp-rosa-backing-up-applications
+
+toc::[]
+
+You can use {oadp-first} with {product-rosa} (ROSA) clusters to back up and restore application data.
+
+ROSA is a fully-managed, turnkey application platform that allows you to deliver value to your customers by building and deploying applications.
+
+ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to speed up the building and delivering of differentiating experiences to your customers.
```

Comment:
Out of curiosity, where are these statements coming from?

Reply:
I received them from someone else.
I think they are coming from here.
https://docs.openshift.com/rosa/rosa_architecture/rosa-understanding.html

Reply:
Ok cool. Dug deeper and found
https://aws.amazon.com/rosa/ and https://aws.amazon.com/products/frontend-web-mobile/

I thought they meant ocp has something mobile related.

Thanks!